docs/labproject.md

```diff
@@ -0,0 +1,39 @@
+# Lab project service
+
+The [lab project](/utonics/labproject/main.go) service takes advantage of almost all the features of Tonic.  Its purpose is to create repositories inside an organisation on GIN where the user is not an administrator or owner.
+It defines the following service components:
+
+## Form
+
+The Form consists of a two pages.
+
+The first page consists of three elements:
+1. Lab organisation: The name of the organisation in which the repository is going to be created.
+2. Project name: The name of the repository or repositories to be created.
+3. Description: A long description for the project.
```

Comment:
```suggestion
1. Lab organisation: The name of the organisation in which the repository is going to be created.
2. Optionally a team name where the repository will be added. If none is indicated, a team with the same name as the repository will be created.
3. Project name: The name of the repository or repositories to be created.Project name: The name of the repository or repositories to be created.
4. Description: A long description for the project.
```

Reply:
or is the documentation related to the existing service and this should be an enhancement, i.e. given in an issue ?

Reply:
No, that's fine.  I can add the field to the current PR, it's a small change.